app/models/form526_submission.rb

```diff
@@ -308,42 +312,29 @@ def perform_ancillary_jobs(first_name)
   def workflow_complete_handler(_status, options)
     submission = Form526Submission.find(options['submission_id'])
     if submission.jobs_succeeded?
-      submission.send_form526_confirmation_email(options['first_name'])
+      Form526ConfirmationEmailJob.perform_async(personalization_parameters(options['first_name']))
       submission.workflow_complete = true
       submission.save
     else
-      submission.send_form526_submission_failed_email(options['first_name'])
+      Form526SubmissionFailedEmailJob.perform_async(personalization_parameters(options['first_name']))
     end
   end
 
-  def send_form526_confirmation_email(first_name)
-    email_address = form['form526']['form526']['veteran']['emailAddress']
-    personalization_parameters = {
-      'email' => email_address,
-      'submitted_claim_id' => submitted_claim_id,
-      'date_submitted' => created_at.strftime('%B %-d, %Y %-l:%M %P %Z').sub(/([ap])m/, '\1.m.'),
-      'first_name' => first_name
-    }
```

Comment:
This block of code was repeated for both `send_form526_confirmation_email` and `send_form526_submission_failed_email`, so I extracted the code into a private method `personalization_parameters`.